content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -428,6 +428,49 @@ transitions a lifecycle stage as follows. Feature gates must function for no les
 is deprecated it must be documented in both in the release notes and the corresponding CLI help.
 Both warnings and documentation must indicate whether a feature gate is non-operational.**
 
+## Deprecating a metric
+
+Each component of the Kubernetes control-plane exposes metrics (usually the
+`/metrics` endpoint), which are typically ingested by cluster administrators.
+Not all metrics are the same: some metrics are commonly used as SLIs or used
+to determine SLOs, these tend to have greater import. Other metrics are more
+experimental in nature or are used primarily in the Kubernetes development
+process.
+
+Accordingly, metrics fall under two stability classes (`ALPHA` and `STABLE`);
+this impacts removal of a metric during a Kubernetes release. These classes
+are determined by the perceived importance of the metric. The rules for
+deprecating and removing a metric are as follows:
+
+**Rule #9: ALPHA metrics have no guarantees and can be removed at any time.**
+
+**Rule #10: STABLE metrics must undergo a deprecation lifecycle prior to removal.**
+
+   * **STABLE metric: 3 releases**
```

Comment:
Suggest a `X releases or X months, whichever is longer` approach (matching GA API lifetimes would make sense to me)

Also, the 12 month guaranteed lifetime for GA APIs looks increasingly insufficient. We have enormous difficulty getting people to move from *beta* APIs to GA APIs in a year, let alone moving from one GA API to another. I think the promise of GA APIs (or "stable" metrics in this case) is effectively *forever*

Reply:
We have a built-in forcing function for migration. We break people but allow them to unbreak themselves for a release to migrate.

Reply:
> We break people but allow them to unbreak themselves for a release to migrate.

That doesn't match my understanding of "stable" or "GA".